### ISSUE TITLE
Keyboard users can’t tab past the Reveal or hide additional items button

### ISSUE THREAD

**Issue by scott-redrup-ou** [tinymce/tinymce#10106]
## 📝 Provide a description of the new feature or improvement
When tabbing through buttons in the toolbar the user becomes stuck once they get to the reveal or hide additional items button when in its hidden state. 

1. Load the [TinyMCE demo](https://www.tiny.cloud/docs/tinymce/latest/full-featured-premium-demo/) 
2. Select the text area and then use ‘Alt + F10’ to access the toolbar 
3. Press tab until you’re stuck on reveal or hide additional items  
4. Notice tabbing does nothing 

When users get to reveal or hide additional items the next item in focus should be the element path. This is consistent with how tabbing works when reveal or additional items is in its revealed state.

### 🫶 What is the motivation?
Currently the experience is confusing for screen reader users who do not receive any audio feedback when trying to tab past reveal or hide additional items and are essentially ‘stuck’. This would also support compliance with WCAG: 2.4.3 Focus Order (Level A). 

### 🔗 What is the consequence of not having this feature?
The experience is confusing for screen reader users. 

### 🚦 How important would you rate the requested feature or improvement?
Important

**Comment by Afraithe** [comment]
We have escalated this internally, this is not intended behavior.

**Comment by MitchC1999** [comment]
This was fixed in [TinyMCE 7.7.0](https://www.tiny.cloud/docs/tinymce/7/7.7.0-release-notes/#keyboard-navigation-would-get-stuck-on-the-more-toolbar-button).
